src/widget/widget.h

```diff
@@ -160,6 +160,9 @@ private slots:
     void processOfflineMsgs();
     void friendListContextMenu(const QPoint &pos);
 
+private:
+    const int icon_size = 15;
```

Comment:
Please move initialization to c'tor so order stays clear. Not every "feature" c++11/14 offers is a good one ...